packages/medusa-js/src/request.ts

```diff
@@ -83,6 +90,13 @@ class Client {
       .join("-")
   }
 
+  isAuthenticated(path, method): boolean {
```

Comment:
Nit: perhaps renaming to `requiresAuthentication` is more descriptive? When I first read the method name, I thought the `isAuthenticated` method checks whether a user is authenticated or not rather than if an endpoint **requires** authentication

Reply:
yes, i agree